docs/community/team.md

```diff
@@ -0,0 +1,18 @@
+# GradleUp Team
+
+The project is led by a small team,
+with a consensus-based decision making and governance.
+There is no special governance charter at the moment.
+
+## Members
+
+- [Alex Nordlund (@deepy)](https://github.com/deepy)
+- [Jean-Michel Fayard (@jmfayard)](https://github.com/jmfayard)
+- [Louis CAD (@LouisCAD)](https://github.com/LouisCAD)
+- [Martin Bonnin (@martinbonnin)](https://github.com/martinbonnin)
+- [Pablisco (@pablisco)](https://github.com/pablisco)
+- [Said Tahsin Dane (@tasomaniac)](https://github.com/tasomaniac)
+
+## Contact the Team
+
+For now, use DMs on the Kotlin Foundation or the Gradle Community Slack.
```

Comment:
Would be cool to have a public `#gradleup`  on the Gradle slack

Reply:
We can create one once we get the trademark usage grant. But yes, this is my assumption too